scan.conf

```diff
@@ -0,0 +1,793 @@
+##
+## Example config file for the Clam AV daemon
+## Please read the clamd.conf(5) manual before editing this file.
+##
+
+
+# Comment or remove the line below.
+#Example
+
+# Uncomment this option to enable logging.
+# LogFile must be writable for the user running daemon.
+# A full path is required.
+# Default: disabled
+#LogFile /var/log/clamd.scan
+LogFile /var/log/mediawiki/clamd.scan
```

Comment:
It looks like the `/var/log/mediawiki` directory is never created and I assume the `clamscan` won't create it for us

Reply:
yes, clamscan does not create it. I added commit to crete the directory https://github.com/WikiTeq/docker-wikiteq-mediawiki/pull/54/commits/69c3cffca2621e38c48a4aee35ccaca4a6866376 and waited when an image will be built to check it,   but it didn't built the image...

Reply:
oh, that's weird, either way, I see the mkdir now, resolved

Reply:
it seems the clamav client writes nothing to the file, I removed this line